docker/Dockerfile.amd

```diff
@@ -45,11 +63,52 @@ ENV PATH=${PATH}:/opt/openmpi/bin:$AOMP/bin
 ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/opt/openmpi/lib:$AOMP/lib
 ENV OMPI_CC=$AOMP/bin/clang
 
-# Devito env
+# Install Omnitrace, an AMDResearch profiling tool akin to Nvidia's Nsight-systems
```

Comment:
I would drop akin to....

Reply:
Omniperf is very similar to Nsight-Compute but the capabilities of Omnitrace far exceed the capabilities of Nsight-Systems. Intel's VTune is closer analogue.